Library/Navigation.swift

```diff
@@ -260,6 +267,33 @@ private let routes: [String:(RouteParams) -> Decoded<Navigation>] = [
   "/users/:user_param/surveys/:survey_response_id": userSurvey
 ]
 
+private let deepLinkRoutes: [String:(RouteParams) -> Decoded<Navigation>] = allRoutes.allValues(
```

Comment:
this is the allow-list of routes we'll use for deep linking. I'm using this lil private helper `Dictionary.allValues(from:)` to pluck out a certain set of keys from the `allRoutes` dictionary.

open to other names!